tests/unit/Math/TanTest.php

```diff
@@ -9,20 +9,23 @@
 
 final class TanTest extends TestCase
 {
+    use FloatAsserts;
+
     /**
      * @dataProvider provideData
      */
-    public function testTan(float $expected, float $number): void
+    public function testTan(float $expected, float $number, float $epsilon = PHP_FLOAT_EPSILON): void
     {
-        static::assertSame($expected, Math\tan($number));
+        static::assertFloatEquals($expected, Math\tan($number), $epsilon);
     }
 
     public function provideData(): array
     {
         return [
             [
                 -3.380515006246586,
-                5.0
+                5.0,
+                0.00000000000001
```

Comment:
it is of by one on my system for some odd reason. Therefor I decided to check only the first 14 digits